net-utils/src/lib.rs

```diff
@@ -470,17 +448,17 @@ pub async fn bind_to_async(
     port: u16,
     reuseport: bool,
 ) -> io::Result<TokioUdpSocket> {
-    let config = SocketConfig::default().reuseport(reuseport);
+    let config = SocketConfig {
+        reuseport,
+        ..Default::default()
+    };
     let socket = bind_to_with_config_non_blocking(ip_addr, port, config)?;
     TokioUdpSocket::from_std(socket)
 }
 
 pub fn bind_to_localhost() -> io::Result<UdpSocket> {
-    bind_to(
```

Comment:
getting rid of calls to bind_to since I want to deprecate it too (as it allows caller to set reuseport without understanding what it does). But doing this needs a separate PR.

Reply:
can this be a separate PR?